mopidy/backend.py

```diff
@@ -277,11 +276,11 @@ def playlists(self):
 
         Read/write. List of :class:`mopidy.models.Playlist`.
         """
-        return copy.copy(self._playlists)
+        raise NotImplementedError
```

Comment:
This would blow up on https://github.com/mopidy/mopidy/blob/develop/mopidy/core/playlists.py#L27

Maybe simply change the default to `return []`?